code/modules/events/radiation_storm.dm

```diff
@@ -12,7 +12,7 @@
 	announceWhen = 1
 
 /datum/round_event/radiation_storm/announce(fake)
-	priority_announce("High levels of radiation detected near the station. Maintenance is best shielded from radiation.", "Anomaly Alert", ANNOUNCER_RADIATION)
+	priority_announce("High levels of radiation detected near the station, you have 60 seconds to take shelter at the Bar and Kitchen, and enjoy the wonderful food and drink the staff have made!", "Anomaly Alert", ANNOUNCER_RADIATION)
```

Comment:
what if they didn't make any

Reply:
We pay them to do their jobs, this sounds like a player problem.